ui/ozone/platform/wayland/wayland_window.h

```diff
@@ -112,6 +117,10 @@ class WaylandWindow : public PlatformWindow, public PlatformEventDispatcher {
   bool has_pointer_focus_ = false;
   bool has_keyboard_focus_ = false;
 
+  bool is_minimized_ = false;
```

Comment:
alternatively we could have a

class enum WindowState { Maximized, Minimized, Fullscreen, Default };

?